docs/index.rst

```diff
@@ -84,17 +84,17 @@ User Documentation
    io/ascii/index
    io/votable/index
    io/misc
+   vo/index
```

Comment:
We can change the title of the section...

But don't you think that with virtual observatory, one mostly "observes" something, i.e., gets data?  I guess missing is the "output" side.

Anyway, it is mostly that I feel it does *not* belong in computations with cosmology and statistics...